requirements-external.txt

```diff
@@ -1,6 +1,6 @@
 emcee
 pyjags
-pymc3 @ git+https://github.com/pymc-devs/pymc3
```

Comment:
This shouldnt change, we should be testing on pymc3 main branch, not the pypi release

Reply:
This won't change where we test on, it only changes the dependency we list here which I think should not be the github version. 

The version used for testing is defined at https://github.com/arviz-devs/arviz/blob/main/.azure-pipelines/azure-pipelines-external.yml#L88

Reply:
bit of a side note which might not be more than my memory failing but I think is relevant. IIRC, this used to be fine (where fine means slow but worked) because the versions of pymc3 last release and pymc3 github were the same, so pip downloaded the code from github, checked the version and saw it was the same installed, so it did not reinstall pymc3. Now the latest release is 3.11.4 but the gitub one is 4.0-dev or somehting like this, so now, we install the "desired" version of pymc3 (3.11.4) and then do `pip install -r requirements-external.txt` which ends up installing 4.0-dev always.